docs/reference-architectures/sap/index.md

```diff
@@ -0,0 +1,418 @@
+    With HANA, use HANA-supported replication solutions such as HSR or Storage Replication. For SQL Server, use AlwaysOn Availability Group replicas.  
+
+## Operational considerations
+SAP HANA has a backup feature that makes use of the underlying Azure
+infrastructure. To back up the SAP HANA database running on Azure virtual
+machines, both the SAP HANA snapshot and Azure storage snapshot are used to
+ensure the backup files’ consistency. For details, see [Backup guide for SAP HANA on Azure Virtual Machines][hana-backup] and the [Azure Backup service FAQ][backup-faq].
+
+Azure provides several functions for [monitoring and diagnostics][monitoring] of the overall infrastructure. Also, enhanced monitoring of Azure virtual
+machines (Linux or Windows) is handled by Azure Operations Management Suite
+(OMS).
+
+To provide SAP-based monitoring of resources and service performance of the SAP
+infrastructure, the Azure SAP Enhanced Monitoring extension is used. This
+extension feeds Azure monitoring statistics into the SAP application for
+operating system monitoring and DBA Cockpit functions. 
+
+## Security considerations
+SAP has its own Users Management Engine (UME) to control role-based access and
+authorization within the SAP application. For details, see [SAP HANA Security - An Overview][sap-security]. (A SAP Service Marketplace account is required for access.)
+
+For infrastructure security, data is safeguarded in transit and at rest. The
+“Security considerations” section of the [SAP NetWeaver on Azure Virtual Machines (VMs) – Planning and Implementation Guide][netweaver-on-azure] begins to address network security. The guide also specifies the network ports
+you must open on the firewalls to allow application communication. 
+
+To encrypt Windows and Linux IaaS virtual machine disks, you can use [Azure Disk Encryption][disk-encryption],
+now in general availability in all Azure public regions and AzureGov regions for
+Standard virtual machines and virtual machines with Premium storage. Azure Disk
+Encryption uses the BitLocker feature of Windows and the DM-Crypt feature of
+Linux to provide volume encryption for the operating system and the data disks.
+The solution also works with Azure Key Vault to help you control and manage the
+disk-encryption keys and secrets in your key vault subscription. Data on the
+virtual machine disks are encrypted at rest in your Azure storage.
+
+For SAP HANA data-at-rest encryption, we recommend using the SAP HANA native
+encryption technology.
+
+> [!NOTE]
+> Don’t use the HANA data-at-rest encryption with Azure disk encryption on the same server.
+> 
+>
+
+
+## Compliance considerations
+Azure as an infrastructure platform is compliant with many security and regulatory requirements. For more information, see the [Azure Trust Center][azure-trust-center]. For guidelines about implementing compliance, see the list of [white papers][white-papers] on the Azure blog. 
```

Comment:
Nothing SAP specific here, this is a general statement about Azure - do we have a specific reason we want to include these?